libraries/botframework-streaming/tests/NamedPipe.test.js

```diff
@@ -1,6 +1,7 @@
 const net = require('net');
 const np = require('../lib');
 const npt = require('../lib/namedPipe/namedPipeTransport');
+const NodeServerUtils = require('../lib/utilities/createNodeServer');

```

Comment:
```suggestion
const { createNodeServer, getNetServerConstructor } = require('../lib/utilities/createNodeServer');

```

Mostly a style nit. Ignore me if you'd like.